main.py

```diff
@@ -176,7 +183,7 @@ def presenters(input_filename: Path, output_folder: Optional[Path] = None):
                 layout="speaker-template",
                 name=row.get("Name"),
                 # override_schedule_title: Optional[str] = None
-                photo_url=row.get("Picture", ""),
+                photo_url=row.get("Picture") or default_profile_pic,
```

Comment:
```suggestion
                photo_url=default_profile_pic or row.get("Picture", ""),
```

We want to use the downloaded one even if there's one from Pretalx